p2p/src/main/kotlin/maru/p2p/Libp2pNetworkFactory.kt

```diff
@@ -108,6 +108,8 @@ class Libp2pNetworkFactory(
         { _ -> 50.0 }, // TODO: I guess we need a scoring function here
       )
 
+    maruPeerManager.setPeerManager(peerManager)
```

Comment:
Can this be provided in the constructor of the `MaruPeerManager` instead? This makes the initialisation of `MaruPeerManager` more complex